confgenerator/testdata/bms/builtin/linux/builtin/golden/f120d4527bd717cab023dbbe5fbdc332.lua

```diff
@@ -0,0 +1,42 @@
+
+function process(tag, timestamp, record)
+local __field_0 = (function()
+return record["agent.googleapis.com/log_file_path"]
+end)();
+local __field_1 = (function()
+if record["logging.googleapis.com/labels"] == nil
+then
+return nil
+end
+return record["logging.googleapis.com/labels"]["compute.googleapis.com/resource_name"]
+end)();
+local __field_2 = (function()
+return record["logging.googleapis.com/logName"]
+end)();
+(function(value)
+record["agent.googleapis.com/log_file_path"] = value
+end)(nil);
+local v = __field_0;
+(function(value)
+if record["logging.googleapis.com/labels"] == nil
+then
+record["logging.googleapis.com/labels"] = {}
+end
+record["logging.googleapis.com/labels"]["agent.googleapis.com/log_file_path"] = value
+end)(v)
+local v = __field_1;
+if v == nil then v = "" end;
```

Comment:
there are a few examples of this in this file. could be simplified to using lua's `or` operator

Reply:
The same applies here. I didn't write these lua files.

Reply:
This is all auto-generated code. I think the code in question comes from [here](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/filter/internal/ast/ast.go#L127).